### ISSUE TITLE
Error if array is stopped

### ISSUE THREAD

**Issue by Portagoras** [dcflachs/compose_plugin#1]
The following warnings appear on the Docker-Tab when the plugin is installed and the Unraid Array is stopped.

`Warning: stream_socket_client(): unable to connect to unix:///var/run/docker.sock (No such file or directory) in /usr/local/emhttp/plugins/dynamix.docker.manager/include/DockerClient.php on line 682
Couldn't create socket: [2] No such file or directory
Warning: Invalid argument supplied for foreach() in /usr/local/emhttp/plugins/dynamix.docker.manager/include/DockerClient.php on line 866`

Nothing too serious, just wanted to let you know.

**Comment by dcflachs** [comment]
What version of unRAID are you using? What version of the plugin? I am unable to reproduce the issue locally, and the paths mentioned are not part of my plugin.

**Comment by Portagoras** [comment]
Thanks for the quick reply :)

Unraid Version: 6.9.2
Plugin Version: 2021.12.03

Maybe it was too quick of a shot. I saw that the paths weren't of your plugin, but of some dynamix plugin, which is probably an internal one, as I do not have that installed.
I noticed the error after installing the compose plugin tho, so I thought I might as well mention it here. I was not able to reproduce it again either, so it probably was something else, as it was a freshly set up server. 

I will bump this if I manage to reproduce the issue. Thanks for looking into it!
